src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -51,7 +51,8 @@
                                         $(RoslynPackagePath)runtimes\**\System.*.dll"
                                Exclude="$(RoslynPackagePath)runtimes\**\System.IO.Pipes.AccessControl.dll"/>
       <!-- binfx folder contains full framework dlls that are needed only on Windows -->
-      <RoslynFrameworkAssembly Include="$(RoslynPackagePath)binfx\**\*" Condition="'$(OS)' != 'Windows_NT'"/>
+      <RoslynFrameworkAssembly Include="$(RoslynPackagePath)binfx\**\*" />
+      <!--<RoslynFrameworkAssembly Include="$(RoslynPackagePath)binfx\**\*" Condition="'$(OS)' != 'Windows_NT'"/> -->
```

Comment:
Whats the intent with the commented out line?

Reply:
Ugg ... that is a misfire. Will fix.